src/features/auth/components/Donation.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { useEffect, useRef, useState } from "react";
+import { Button } from "components/ui/Button";
+import { InnerPanel, OuterPanel, Panel } from "components/ui/Panel";
+import upArrow from "assets/icons/arrow_up.png";
+import downArrow from "assets/icons/arrow_down.png";
+import question from "assets/icons/expression_confused.png";
+import leftArrow from "assets/icons/arrow_left.png";
+import rightArrow from "assets/icons/arrow_right.png";
+import { useLazyEffect } from "lib/utils/useLazyEffect";
+import classnames from "classnames";
+
+const roundToOneDecimal = (number: number) => Math.round(number * 10) / 10;
+export enum CharityAddress {
+  TheWaterProject = "0x060697E9d4EEa886EbeCe57A974Facd53A40865B",
+  Heifer = "0xD3F81260a44A1df7A7269CF66Abd9c7e4f8CdcD1",
+  CoolEarth = "0x3c8cB169281196737c493AfFA8F49a9d823bB9c5",
+}
+interface Charity {
+  name: string;
+  info: string;
+  url: string;
+  address: CharityAddress;
+}
+
+const CHARITIES = [
+  {
+    name: "The Water Project",
+    info: "You can provide clean, safe and reliable water today.",
+    url: "https://thewaterproject.org/donate-ethereum",
+    address: CharityAddress.TheWaterProject,
+  },
+  {
+    name: "The Heifer Project",
+    info: "We do more than train farmers. We grow incomes.",
+    url: "https://www.heifer.org/give/other/digital-currency.html",
+    address: CharityAddress.Heifer,
+  },
+  {
+    name: "Cool Earth",
+    info: "Aim to halt deforestation and its impact on climate change.",
+    url: "https://www.coolearth.org/cryptocurrency-donations/",
+    address: CharityAddress.CoolEarth,
+  },
+];
```

Comment:
Consider adding a random shuffle on these so they are displayed in a different order. This will ensure donations are spread evenly :) 

E.g.
`.sort(() => 0.5 - Math.random());`